### PR TITLE
TDR-1596 - Add includeTopLevelFolder in the consignment

### PR DESCRIPTION
**Pull request by vimleshtna**
Add includeTopLevelFolder in the consignment table while adding file upload details
Update consignment mutation to accept the includeTopLevelFolder

### CODE REVIEW

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala [hunk 1]
```diff
@@ -757,14 +757,15 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
     response.errors.head.extensions.get.code should equal("NOT_AUTHORISED")
   }
 
-  "startUpload" should "add the upload status and update the parent folder" in withContainers { case container: PostgreSQLContainer =>
+  "startUpload" should "add the upload status and update the parent folder and 'IncludeTopLevelFolder' fields" in withContainers { case container: PostgreSQLContainer =>
```
Comment (TomJKing) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala:760]: Maybe reword test slightly to: `... upload status, update the parent folder and ...`

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some minor comments. Thanks

**Comment by TomJKing** [comment]
@vimleshtna All looks OK, but I think you've got a failing unit test.

**Comment by vimleshtna** [comment]
> @vimleshtna All looks OK, but I think you've got a failing unit test.

Yeah, I reran and all tests passed now

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks
